js/content-filters.js

```diff
@@ -0,0 +1,78 @@
+import {debounce} from './functions.js';
+
+const FiltersId = {
+  DEFFAULT: '#filter-default',
+  RANDOM : '#filter-random',
+  DISCUSSED: '#filter-discussed'
+};
+const FILTERS_COUNT = 3;
+const DEFFAULT_SHOWN_MINIATURES = 25;
+const ACTIVE_SELECTOR = 'img-filters__button--active';
+
+const filterContainer = document.querySelector('.img-filters');
+const filterForm = filterContainer.querySelector('.img-filters__form');
+const picturesContainer = document.querySelector('.pictures');
+
+const clearMiniatures = () => {
+  const pictures = picturesContainer.querySelectorAll('.picture');
+  for (let i = 0; i < pictures.length; i++) {
+    pictures[i].parentNode.removeChild(pictures[i]);
+  }
+};
+
+const checkActiveFilter = (id) =>
+  filterForm.querySelector(id).classList.contains(ACTIVE_SELECTOR);
+
+const addActiveFilterClass = (id) => {
+  for (let i = 0; i < FILTERS_COUNT; i++) {
+    const filter = filterForm.children[i];
+    if (filter.classList.contains(ACTIVE_SELECTOR)) {
+      filter.classList.remove(ACTIVE_SELECTOR);
+    }
+  }
+  filterForm.querySelector(id).classList.add(ACTIVE_SELECTOR);
+};
+
+const createFilteredMiniatures = (photosWithDescriptions, renderMiniatures, clearMiniaturesDebounced, id, sortCb = false, numberToShow = DEFFAULT_SHOWN_MINIATURES) => {
```

Comment:
```suggestion
const createFilteredMiniatures = (photosWithDescriptions, renderMiniatures, clearMiniaturesDebounced, id, sortCb = false, numberToShow = DEFFAULT_SHOWN_MINIATURES) => {
```

Давай уберем дефолтное значение для параметра `sortCb` так как мы ожидаем функцию, а дефолтное значение - булевое